WordPress/src/main/res/layout-sw600dp/theme_grid_cardview_header.xml

```diff
@@ -18,10 +18,12 @@
             android:layout_height="match_parent">
 
             <LinearLayout
+                android:id="@+id/header_card_left_side_linear_layout_container"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_alignParentStart="true"
                 android:layout_margin="@dimen/margin_large"
+                android:layout_toStartOf="@id/header_card_right_side_linear_layout_container"
```

Comment:
Warning (⚠️): This actually changes the layout. A solution would be to rewrite that specific part of the layout using another approach, or completely using `ConstraintLayout`. In the meanwhile I suggest just ignoring it for now using, wdyt?

```
tools:ignore="RelativeOverlap"
```